website/more_documentation/button_documentation.py

```diff
@@ -33,3 +38,21 @@ async def await_button_click() -> None:
             ui.label('Two')
             await b.clicked()
             ui.label('Three')
+
+    @text_demo('Disable button with a context manager', '''
+        This showcases a async context manager that can be used to disable a button for the duration of an async process.
+    ''')
+    async def disable_context_manager() -> None:
+        @asynccontextmanager
+        async def disable(element: DisableableElement) -> None:
```

Comment:
The correct return type annotation should be `Generator[None, None, None]`, I guess.

Reply:
Chat GPT suggested `ContextManager[None]` as a slightly better option. But I think the return types were fairly pointless and cluttering up the example so I just removed them.